mindsdb/interfaces/database/database.py

```diff
@@ -69,6 +69,17 @@ def get_dict(self, company_id: Optional[int], filter_type: Optional[str] = None)
             for x in self.get_list(company_id=company_id, filter_type=filter_type)
         )
 
+    def get(self, integration_id, company_id=None):
+        # TODO get directly from db?
```

Comment:
Please:
1. provide docstrings in google styleguide format https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings.
2. provide type of input parameters. you may do in direct in function signature or in the doc strings.

Reply:
added docstring